demos/python_demos/action_recognition/README.md

```diff
@@ -45,32 +46,33 @@ Options:
   -h, --help            Show this help message and exit.
   -i INPUT, --input INPUT
                         Required. An input to process. The input must be a single image,
-                        a folder of images or anything that cv2.VideoCapture can process
-  --loop                Optional. Enable reading the input in a loop
+                        a folder of images or anything that cv2.VideoCapture can process.
+  --loop                Optional. Enable reading the input in a loop.
+  -o OUTPUT, --output OUTPUT
```

Comment:
Do we define somewhere what will happens in case of endless input, like input from camera or when '-loop' option is active? We might need some safety guard for that potentially dangerous option. Like, by default store only first 100 or 1000 frames and with adding additional parameter, '-output_limit' allow to set -1 to store all the input frames and positive number to specify how many frames to store (or even the range of frames to store, like 100,-1 - to store from 100-th frame up to the end of input stream.

Reply:
I assume it will fill all available disk space with output. But that's what one requests by running a demo for long enough. And it doesn't seem to be dangerous. At least no more dangerous then fully taking any other system resource, RAM for example, which we don't handle explicitly as well. My opinion is that we shouldn't extend functionality without actual need.

Reply:
this is dangerous for system stability and such action should requested by user explicitly rather then be side effect. this is an actual need.

Reply:
I still can't see how it differs from RAM, but ok.